_episodes/09-cmorization.md

```diff
@@ -141,38 +147,52 @@ name that was created to store the raw observation data files, i.e.
 If everything is okay, the output should look something like this:
 
 ~~~
-...
-... Starting the CMORization Tool at time: 2022-07-26 14:02:16 UTC
+... Writing program log files to:
+/scratch/b/b309059/esmvaltool_output/data_formatting_20240527_132448/run/main_log.txt
+/scratch/b/b309059/esmvaltool_output/data_formatting_20240527_132448/run/main_log_debug.txt
+... Starting the CMORization Tool at time: 2024-05-27 13:24:48 UTC
 ... ----------------------------------------------------------------------
-... input_dir  = /home/peter/data/RAWOBS
-... output_dir = /home/peter/esmvaltool_output/data_formatting_20220726_140216
+... input_dir  = /work/bd0854/DATA/ESMValTool2/RAWOBS
+... output_dir = /scratch/b/b309059/esmvaltool_output/data_formatting_20240527_132448
```

Comment:
It is not clear if b309059 is a user name. I suggest changing this to username and mentioing that this will be substituted by the individual's username (maybe at the start of the episode).